kotlin-checks-test-sources/src/main/kotlin/checks/PropertyGetterAndSetterUsageCheckSample.kt

```diff
@@ -19,14 +21,172 @@ open class PropertyGetterAndSetterUsageCheckSample {
+        // 1) Setter with visibility <= getter visibility
+
+        // public getter / public setter
+        private var f1: Int = 0
+        fun getF1(): Int = f1 // Noncompliant
+        fun setF1(value: Int) { f1 = value }
+
+        // public getter / internal setter
+        private var f2: Int = 0
+        fun getF2(): Int = f2 // Noncompliant
+        internal fun setF2(value: Int) { f2 = value }
+
+        // public getter / protected setter
+        private var f3: Int = 0
+        fun getF3(): Int = f3 // Noncompliant
+        protected fun setF3(value: Int) { f3 = value }
+
+        // public getter / private setter
+        private var f4: Int = 0
+        fun getF4(): Int = f4 // Noncompliant
+        private fun setF4(value: Int) { f4 = value }
+
+        // internal getter / internal setter
+        private var f5: Int = 0
+        internal fun getF5(): Int = f5 // Noncompliant
+        internal fun setF5(value: Int) { f5 = value }
+
+        // internal getter / private setter
+        private var f6: Int = 0
+        internal fun getF6(): Int = f6 // Noncompliant
+        private fun setF6(value: Int) { f6 = value }
+
+        // protected getter / protected setter
+        private var f7: Int = 0
+        protected fun getF7(): Int = f7 // Noncompliant
+        protected fun setF7(value: Int) { f7 = value }
+
+        // protected getter / private setter
+        private var f8: Int = 0
+        protected fun getF8(): Int = f8 // Noncompliant
+        private fun setF8(value: Int) { f8 = value }
+
+        // private getter / private setter
+        private var f9: Int = 0
+        private fun getF9(): Int = f9 // Noncompliant
+        private fun setF9(value: Int) { f9 = value }
+
+        // ______________________________________________
+        // 2) Setter with visibility > getter visibility
+
+        // internal getter / public setter
+        private var f10: Int = 0
+        internal fun getF10(): Int = f10 // Compliant
+        public fun setF10(value: Int) { f10 = value }
+
+        // internal getter / protected setter
+        private var f11: Int = 0
+        internal fun getF11(): Int = f11 // Compliant
+        protected fun setF11(value: Int) { f11 = value }
+
+        // protected getter / public setter
+        private var f12: Int = 0
+        protected fun getF12(): Int = f12 // Compliant
+        public fun setF12(value: Int) { f12 = value }
+
+        // protected getter / internal setter
+        private var f13: Int = 0
+        protected fun getF13(): Int = f13 // Compliant
+        internal fun setF13(value: Int) { f13 = value }
+
+        // private getter / public setter
+        private var f14: Int = 0
+        private fun getF14(): Int = f14 // Compliant
+        public fun setF14(value: Int) { f14 = value }
+
+        // private getter / internal setter
+        private var f15: Int = 0
+        private fun getF15(): Int = f15 // Compliant
```

Comment:
The getter has the same visibility as the property -> we don't need it.

Reply:
It will be done later by [SONARKT-308](https://sonarsource.atlassian.net/jira/software/c/projects/SONARKT/issues/SONARKT-308)